6_3_iterable.js

```diff
@@ -0,0 +1,25 @@
+class Group {
+    constructor(arr) {
+        this.cluster = arr;
+    }
+    add = value => (this.cluster.includes(value) ? null : this.cluster.push(value));
+    delete = value =>
+        this.cluster.includes(value) ? this.cluster.splice(this.cluster.indexOf(value), 1) : null;
+    has = value => this.cluster.some(item => item === value);
```

Comment:
You have all these `add`, `delete` and `has` methods already declared in the `6-2-group.js` file. It would be nice to extract the declaration of these methods to separate variables and reuse those in both classes/files.